Common/OJS.Common/Utils/EnvironmentUtils.cs

```diff
@@ -43,11 +43,19 @@ public static string GetLoggerFilePath<TProjectStartUp>()
 
         public static void ValidateEnvironmentVariableExists(
             IEnumerable<string> configurationValues)
-            => ValidateEnvironmentVariableExists(configurationValues, GetByKey);
+        {
+            var requiredGlobalConfigValues = new List<string>
+            {
+                DistributorBaseUrlKey,
+                ApplicationUrl,
+                RedisConnectionString,
+                SharedAuthCookieDomain,
+            };
```

Comment:
These should be passed from outside, as this is just Utility method (no business logic). Bring back the ValidateLaunchSettings extension and make it a common one. 
```
public static IServiceCollection ValidateLaunchSettings(this IServiceCollection services, params string[] configurationValues)
{
  var requiredGlobalConfigValues = new List<string>
            {
                DistributorBaseUrlKey,
                ApplicationUrl,
                RedisConnectionString,
                SharedAuthCookieDomain,
            };

   requiredGlobalConfigValues.AddRange(configurationValues);

  EnvironmentUtils.ValidateEnvironmentVariableExists(requiredConfigValues);

}
```

Reply:
Refactored.